docs/native-modules-android.md

```diff
@@ -22,11 +24,11 @@ To get started, open up the Android project within your React Native application
 
 We recommend using Android Studio to write your native code. Android studio is an IDE built for Android development and using it will help you resolve minor issues like code syntax errors quickly.
 
-We also recommend enabling [Gradle Daemon](https://docs.gradle.org/2.9/userguide/gradle_daemon.html) to speed up builds as you iterate on Java code.
+We also recommend enabling [Gradle Daemon](https://docs.gradle.org/2.9/userguide/gradle_daemon.html) to speed up builds as you iterate on Java/Kotlin code.
 
 ### Create A Custom Native Module File
 
-The first step is to create the CalendarModule.java Java file inside `android/app/src/main/java/com/your-app-name/` folder. This Java file will contain your native module Java class.
+The first step is to create the (CalendarModule.java or CalendarModule.kt) Java/Kotlin file inside `android/app/src/main/java/com/your-app-name/` folder. This Java/Kotlin file will contain your native module Java/Kotlin class.
```

Comment:
```suggestion
The first step is to create the (`CalendarModule.java` or `CalendarModule.kt`) Java/Kotlin file inside `android/app/src/main/java/com/your-app-name/` folder. This Java/Kotlin file will contain your native module Java/Kotlin class.
```

Reply:
Plus here there could be confusion on the folder as there is Java in this path `android/app/src/main/java/com/your-app-name/`.

Maybe adding a `(the folder is the same for both for either Kotlin of Java)` would help clarify.